apps/keyboard/test/marionette/lib/keyboard.js

```diff
@@ -97,28 +137,25 @@ Keyboard.prototype = {
     return 2;
   },
 
-  switchCase: function switchCase(upperCase) {
-    if (this.isUpperCase() === upperCase) {
-      return;
-    }
+  getSuggestionKey: function(word) {
+    var selector = Keyboard.Selector.suggestionKey.replace(/%s/, word);
+    return this.client.findElement(selector);
+  },
 
-    var shiftKey = this.shiftKey;
+  getCurrentKeyboard: function() {
+    var activeFrame = this.activeKeyboardFrame;
 
-    shiftKey.tap();
-    this.client.waitFor(function() {
-      var expected = upperCase ? 'true' : 'false';
-      return (shiftKey.getAttribute('aria-pressed') === expected);
-    });
+    return activeFrame.getAttribute('data-frame-name');
```

Comment:
Please change the comment here

https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/input_window.js#L35-L36

to be more descriptive like

> // Set the ID onto the element so Gij tests can probe it with Keyboard#getCurrentKeyboard.